_includes/mailchimp.html

```diff
@@ -0,0 +1,18 @@
+<!-- Begin MailChimp Signup Form -->
+<div id="mc_embed_signup">
+	<form action="//thefirehoseproject.us11.list-manage.com/subscribe/post?u=48d53d72a28bb1c0fca326b5a&amp;id=953d41e871" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" novalidate>
```

Comment:
@Sabotnik @kenmazaika I'm guessing this is where the accounts settings for the official firehose community email account will need to be placed otherwise the code should be the same.